### ISSUE TITLE
Make updates to openworm_docs based on C. Volko's changes

### ISSUE THREAD

**Issue by slarson** [openworm/openworm_docs#51]
Page "Introduction":

<<Then, please browse our project list to understand the different areas where work is happening. To put the projects in context, you will find it useful to read more about the big picture idea of the [modeling approach](modeling) we are taking.>>

I would use "modeling approach".

<<We also have a lot of Java and Javascript/HTML/CSS development going on as part of Geppetto.>>

JavaScript

<<Please reach out to us at info@openworm.org to discuss opportuntities with OpenWorm.>>

opportunities

Page "Background":

<<Tools like algebra and calculus and Newtonian mechanics and computer spreadsheets were advances because we could plug numbers into equations and get answers out that told us something about the world.>>

I would write: "Tools like algebra, calculus, Newtonian mechanics and computer spreadsheets were advances because we could plug numbers into equations and get answers out that told us something about the world."

Page "History":

<<1999: Ferree and Lockery publish a neural network model of chemotaxis in C.Elegans>>

This should be: "1999: Ferree and Lockery publish a neural network model of chemotaxis in C. elegans."

<<2009: Boyle & Cohen publish an integrated 2D computer model of C. elegans locomotion>>

A period is missing at the end of the sentence.

<<24 Jan 2011: Giovanni discovers CyberElegans, developed by Andrey Palyanov and Sergey Khayrulin (from AP Ershov Institute of Informatics, Russia) who are contacted and immediately agree to join forces with the OpenWorm team:>>

This should be: "24 Jan 2011: Giovanni discovers CyberElegans, developed by Andrey Palyanov and Sergey Khayrulin (from AP Ershov Institute of Informatics, Russia), who are contacted and immediately agree to join forces with the OpenWorm team:"

<<Credit to Eugenio Battaglia for the story on his blog of the early OpenWorm history>>

A period is missing at the end of the sentence.

Page "Releases":

<<Release 5,6,7 (2nd Half of 2013, 1st half of 2014, 2nd half of 2014)>>

This should be: "Release 5,6,7 (2nd Half of 2013, 1st Half of 2014, 2nd Half of 2014)"

<<This release, among many acheivements, we accomplished the following:>>>

achievements

<<Our second release pointed us in a good direction for the future, and provided some [http://browser.openworm.org exciting products].>>

This should be: "Our second release pointed us in a good direction for the future, and provided some exciting products [http://browser.openworm.org]."

<<The user will be able to contribute to a shared knowledge space of the positions and identities of c. elegans synapses using an installation of CATMAID. This is important because the c. elegans connectome does not currently incorporate synapse positions at all.>>

C. elegans with a capital letter (twice)

<<EPIC-7: As a user, I want to see a WebGL visualization of Smoothed Particle Hydrodynamics -----------------------------------------------------------------------------------------We want to be able to run the Smoothed Particle Hydrodynamics demos so we can see them through the browser.>>

The formatting of this passage is not correct.

<<Our major goal for this release was to integrate the work we have done in release one to do a detailed simulation of a body wall muscle cell, MDL08 (pictured below).>>

Picture is missing.

EPIC-1 ... EPIC-8: Sometimes the sentences end with a period and sometimes they do not, please me consistent.

<<EPIC-8 As an open worm team member, I want to launch a fundraising campaign to raise money for the project>>

Open Worm should be written with capital letters.

Paragraphs "Simulation Engine", "Worm Browser": Remove the periods.

Page "Modelling":

<<Our main goal is to build the world's first virtual organism-- an in silico implementation of a living creature-- for the purpose of achieving an understanding of the events and mechanisms of living cells.>>

This should be: "Our main goal is to build the world's first virtual organism -- an in silico implementation of a living creature -- for the purpose of achieving an understanding of the events and mechanisms of living cells."

<<While our ultimate goal is to simulate every cell in the c. elegans, we are starting out by building a model of its body and environment, its nervous system, and its muscle cells.>>

C. elegans with capital letter.

<<In this movie you can see a simulated 3D c. elegans being activated in an environment.>>

Same as above.

<<For an excellent review of the current state of research on this topic, check out Cohen & Sanders, 2014>>

Missing period.

<<In this movie you can see a simulated 3D c. elegans being activated in an environment.>>

C. elegans with capital letter.

<<More detailed information is available on the Sibernetic project page \<sibernetic-project>.>>

Link is missing. I guess the correct link should be: http://docs.openworm.org/en/0.9/Projects/sibernetic/

<<Our computational strategy to model the nervous system involves first reusing the c. elegans connectome and the 3D anatomical map of the c. elegans nervous system and body plan. We have used the NeuroML standard (Gleeson et al., 2010) to describe the 3D anatomical map of the c. elegans nervous system. This has been done by discretizing each neuron into multiple compartments, while preserving its three-dimensional position and structure. We have then defined the connections between the NeuroML neurons using the c. elegans connectome.>>

C. elegans with capital letter.

Page "Sibernetic":

<<While our ultimate goal is to simulate every cell in the c. Elegans, we are starting out by building a model of its body, its nervous system, and its environment.>>

C. elegans.

<<In this movie you can see a simulated 3D c. elegans being activated in an environment.>>

Same as above.

<<@vellamike, @a-palyanov and @skhayrulin are taking the lead on this,>>

Period instead of comma.

<<Circa 2010 Neuromechanical model of C. Elegans>>

C. elegans.

Page "Geppetto":

<<Geppetto, is written in Java and leverages technologies like OSGi, Spring Framework, OpenCL and Maven.>>

This should be: "Geppetto is written in Java and leverages technologies like OSGi, Spring Framework, OpenCL and Maven."

Page "Movement":

<<The main goal of the Movement Validation team is to finish a test pipeline so the OpenWorm project can run a behavioural phenotyping of its virtual worm, using the same statistical tests the Schafer lab used on their real worm data.>>

behavioral

<<This means building a training set and a test set that are kept separate from each other, using the training set to tune up the model, then generating predictions, and comparing them against the test set, and doing some cross-validation).>>

This should be: "This means building a training set and a test set that are kept separate from each other, using the training set to tune up the model, then generating predictions, and comparing them against the test set, and doing some cross-validation."

<<All issues related to movement validation can be found on GitHub>>

Missing period.

Page "Representation":

<<The c. elegans connectome (wiring diagram of neurons)
Cell list of c. elegans
Ion channels used by c. elegans>>

C. elegans.

<<Below you can find information about each subproject, see the project’s current roadmap and access the associated data repositories>>

Period missing.

<<A lot of data about c. elegans is integrated into the model.>>

C. elegans.

<<We have used the NeuroML standard (Gleeson et al., 2010) to describe the 3D anatomical map of the c. elegans nervous system.>>

C. elegans.

<<We have then defined the connections between the NeuroML neurons using the c. elegans connectome.>>

C. elegans.

<<The NeuroML connectome model provides a framework for multi-compartmental modeling of the c. elegans nervous system.>>

C. elegans.

<<Create or reuse a NeuroML description of c. elegans motor neuron synapses>>

C. elegans.

Page "Muscle / Neuron / Channel":

<<Filling the database with information, being sure to tag each fact with sources along the way.>>

Remove period.

Page "Community Overview":

<<An example of an effective proposal is available online>>

Period is missing.

<<Compliance can be checked with the pep8 tool and autopep8>>

Period is missing.

Page "Resources":

<<You can download the binary for the CyberElegans (Windows only)>>

Period is missing.

<<This is built from the Virtual Worm Blender files>>

Period is missing.

<<Explore the c. elegans in 3D! The WormBrowser is an interactive virtual experience of browsing the C. elegans worm anatomy. This is built from the Virtual Worm Blender files>>

C. elegans. At end of last sentence, period is missing.

<<More information about working with the data within it and other data entities can be found on the data representation project page \<data-rep>.>>

Link missing. I guess the correct link would be: http://docs.openworm.org/en/0.9/Projects/datarep/

Page "FAQ":

<<it is a little unintuitive but that's why makes up how it "thinks".>>

Sentence should start with capital letter.

<<Given all that we DON'T know about c. elegans>>

C. elegans.

<<Is there only one solution to all those variables in the connectome that will make a virtual c. elegans that resembles a real one, or are there multiple?>>

C. elegans.

<<The community of scientists using c. elegans as their model organism is much larger than communities that studying simpler multi-cellular organisms, so the effect of the community size also weighed in on the decision.>>

C. elegans.

<<You can read more about this here>>

Period is missing.

<<The degree of overlap between the simulated worm and the behavior of real worms will be very interesting to see --we are very curious to find this out!>>

Should be: "The degree of overlap between the simulated worm and the behavior of real worms will be very interesting to see -- we are very curious to find this out!"

<<The c. elegans connectome (wiring diagram of neurons)
Cell list of c. elegans
Ion channels used by c. elegans>>

C. elegans.

<<There have been other modeling efforts in C. Elegans and their subsystems, as well as in academic journal articles.>>

C. elegans.

<<Here are some examples>>

Period is missing.

<<Then we will approximate the six motor neurons synapsing onto it based on what we know about its ion channels and whatever more we can gather based on calcium imaging.Then we will be exploring how to tune the combined system of the single muscle cell with the 6 motor neurons connected to it as a network and radiate outwards from there.>>

Missing space after the period ending the first sentence.

<<We have encoded information about the neurons in the OpenWorm NeuroML spatial connectome>>

Period is missing.

<<We both want to see the c. elegans reverse engineered as a means of understanding nervous systems.>>

C. elegans.

**Comment by slarson** [comment]
Thanks to @adokhugi for these changes!
